docs/_guide/02-writing-templates.md

```diff
@@ -84,7 +84,7 @@ There are a few types of bindings:
     ```
   * Boolean Attribute:
     ```js
-    html`<input type="checkbox" ?checked=${checked}>`
+    html`<button type="submit" ?disabled=${disabled}>`
```

Comment:
Should include closing tag, or stick with self-closing `<input>` instead.

Reply:
sure, I'll swap it to a disabled input